xbmc/cores/AudioEngine/Sinks/AESinkStarfish.cpp

```diff
@@ -159,12 +161,28 @@ bool CAESinkStarfish::Initialize(AEAudioFormat& format, std::string& device)
     return false;
   }
 
+  m_acbId = AcbAPI_create();
```

Comment:
This a little hack. The shim library will return `0` on webOS 5+ here, that's why the it doesn't break. Let's see first if it indeed fixes the passthrough sink, then come up with a better way to detect acb presence